src/core/components/hv-route/index.tsx

```diff
@@ -472,6 +472,7 @@ function RouteFC(props: Types.FCProps) {
           getState: () => docContext.getState(),
           registerPreload: (id: number, el: Element) =>
             props.setPreload(id, el),
+          reload: () => {},
```

Comment:
can't we also hook the `load` function here?

Reply:
Reworked the needsLoad to use a state which will trigger the inner class component reload.